src/main/java/com/sprint/mission/discodeit/controller/ReadStatusController.java

```diff
@@ -0,0 +1,48 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.message.ReadStatusCreateDto;
+import com.sprint.mission.discodeit.dto.message.ReadStatusUpdateDto;
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.service.ReadStatusService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/read-statuses")
+public class ReadStatusController {
+
+    private final ReadStatusService readStatusService;
+
+    @Autowired
+    public ReadStatusController(ReadStatusService readStatusService) {
+        this.readStatusService = readStatusService;
+    }
```

Comment:
다른 컨트롤러와 일관성을 위해 `@Autowired`는 제거해도 될 것 같아요.
```suggestion
    public ReadStatusController(ReadStatusService readStatusService) {
        this.readStatusService = readStatusService;
    }
```